src/com/oltpbenchmark/OLTPBench.java

```diff
@@ -0,0 +1,190 @@
+            options.addOption((String)null, "clear", true, "Clear all records in the database for this benchmark");
+            options.addOption((String)null, "load", true, "Load data using the benchmark's data loader");
+            options.addOption((String)null, "execute", true, "Execute the benchmark workload");
+            options.addOption((String)null, "runscript", true, "Run an SQL script");
+            options.addOption((String)null, "upload", true, "Upload the result");
+            options.addOption("v", "verbose", false, "Display Messages");
+            options.addOption("h", "help", false, "Print this help");
+            options.addOption("s", "sample", true, "Sampling window");
+            options.addOption("im", "interval-monitor", true, "Throughput Monitoring Interval in milliseconds");
+            options.addOption("ss", false, "Verbose Sampling per Transaction");
+            options.addOption("o", "output", true, "Output file (default System.out)");
+            options.addOption("d", "directory", true, "Base directory for the result files, default is current directory");
+            options.addOption("t", "timestamp", false, "Each result file is prepended with a timestamp for the beginning of the experiment");
+            options.addOption("ts", "tracescript", true, "Script of transactions to execute");
+            options.addOption((String)null, "histograms", false, "Print txn histograms");
+            options.addOption((String)null, "dialects-export", true, "Export benchmark SQL to a dialects file");
+            CommandLine argsLine = parser.parse(options, args);
+            if (argsLine.hasOption("h")) {
+                oltpMethods.printUsage(options);
+            } else if (!argsLine.hasOption("c")) {
+                LOG.error("Missing Configuration file");
+                oltpMethods.printUsage(options);
+            } else if (!argsLine.hasOption("b")) {
+                LOG.fatal("Missing Benchmark Class to load");
+                oltpMethods.printUsage(options);
+            } else {
+                int intervalMonitor = 0;
+                if (argsLine.hasOption("im")) {
+                    intervalMonitor = Integer.parseInt(argsLine.getOptionValue("im"));
+                }
+
+                String targetBenchmarks = argsLine.getOptionValue("b");
+                String[] targetList = targetBenchmarks.split(",");
+                List<BenchmarkModule> benchList = new ArrayList();
+                List<TransactionType> activeTXTypes = new ArrayList();
+                String configFile = argsLine.getOptionValue("c");
+                XMLConfiguration xmlConfig = new XMLConfiguration(configFile);
+                xmlConfig.setExpressionEngine(new XPathExpressionEngine());
+                System.out.println(oltpMethods.pluginConfigBenchmark(targetList, xmlConfig, argsLine, pluginConfig, configFile, activeTXTypes, benchList));
```

Comment:
Shouldn't this be `LOG.info`?

Reply:
I don't have necessary knowledges how works 'Log.info' however I'll be study its functionality.